### DISCUSSION TITLE
MFA in Community Edition

### DISCUSSION THREAD

**Discussion by markusglaetzner** [shellhub-io/shellhub#2917] [Ideas]
An application that can be used to access protected networks should also be provided with maximum security in a community edition. In my view, this requires at least a 2FA when accessing the web interface. In the age of fishing, username and password are no longer a security criterion. Without MFA the application is not sufficiently protected and thus unfortunately not seriously usable even in a community edition.

**Comment by gustavosbarreto** [comment]
Following our open-source business model, we have just added support for two-factor authentication (2FA) using TOTP in our latest Enterprise and Cloud editions.
